docs/source/Internals.md

```diff
@@ -113,11 +113,12 @@ field to be read.
 
 ### Strings and Vectors
 
-Strings are simply a vector of bytes, and are always
-null-terminated. Vectors are stored as contiguous aligned scalar
-elements prefixed by a 32bit element count (not including any
-null termination). Neither is stored inline in their parent, but are referred to
-by offset.
+Strings are simply a vector of bytes, and are always null-terminated. Vectors
+are stored as contiguous aligned scalar elements prefixed by a 32bit element
+count (not including any null termination). Neither is stored inline in their
+parent, but are referred to by offset. FlatBuffers does not deduplicate
```

Comment:
FlatBuffers does not de-duplicate strings either, that is an API that the user may or may not call (e.g. `CreateSharedString`). You can even share vectors, but you'd have to track your own duplicates.

So from the pov of `Internals.md` which is all about the wire format (and not the API that creates it), it is more correct to say that table/vector elements that are encoded by an offset (which may be tables, strings, vectors) are potentially shared/duplicates, if the user explicitly serializes the same offset twice.

Reply:
Right. I was assuming that de-duplication was a property of the encoding (i.e. binary wire format vs JSON) rather than on the API, mainly given I've been only using Flat/FlexBuffers through `flatc`. I'll improve the PR accordingly.